### ISSUE TITLE
api with "null" value for "api_shortname" is breaking label sync

### ISSUE THREAD

**Issue by bcoe** [googleapis/repo-automation-bots#632]
This logic:

```js
crypto_1.createHash('md5')
                .update(api.api_shortname)
                .digest('hex')
                .slice(0, 6),
```

Is breaking in label sync, as there appears to be an `api` with a `null`, `api_shortname`.

**Comment by bcoe** [comment]
Here are the culprits from the logs:

```
A 2020-06-30T01:42:42.107Z label_sync gci3gtybavlj err with { display_name: 'Apps Script', label_sync gci3gtybavlj 
A 2020-06-30T01:42:42.107Z label_sync gci3gtybavlj   github_label: 'product: apps_script', label_sync gci3gtybavlj 
A 2020-06-30T01:42:42.107Z label_sync gci3gtybavlj   api_shortname: null } label_sync gci3gtybavlj 
A 2020-06-30T01:42:42.107Z label_sync gci3gtybavlj err with { display_name: 'Data Studio', label_sync gci3gtybavlj 
A 2020-06-30T01:42:42.107Z label_sync gci3gtybavlj   github_label: 'product: datastudio', label_sync gci3gtybavlj 
A 2020-06-30T01:42:42.107Z label_sync gci3gtybavlj   api_shortname: null } label_sync gci3gtybavlj 
````

**Comment by JustinBeckwith** [comment]
I think I fixed this in #692
